runtime/queries/zig/injections.scm

```diff
@@ -0,0 +1,5 @@
+[
+  (container_doc_comment)
+  (doc_comment)
+  (line_comment)
+] @comment
```

Comment:
I'm not sure that this would work, since you also have to specify the injection language like here: https://github.com/helix-editor/helix/blob/master/runtime/queries/rust/injections.scm

Reply:
I just remove them for now, does not really matter.